app/lib/theme/ObjectFieldTemplate.tsx

```diff
@@ -1,40 +1,104 @@
-import Grid from '@button-inc/bcgov-theme/Grid';
 import FormBorder from './components/FormBorder';
+import styled from 'styled-components';
+import { ObjectFieldTemplateProps } from '@rjsf/core';
 
 const DefaultDescriptionField = (props: {
   id: string;
   description: string;
 }) => <span id={props.id}>{props.description}</span>;
 
-const ObjectFieldTemplate = (props: any) => {
+const StyledInline = styled('div')`
+  display: flex;
+  input {
+    min-width: 90%;
+  }
+`;
+
+const StyledFull = styled('div')`
+  input {
+    min-width: 50%;
+  }
+`;
+
+const StyledLabel = styled('div')`
+  margin-bottom: 4px;
+`;
+
+const ObjectFieldTemplate = (props: ObjectFieldTemplateProps) => {
   const DescriptionField = props.DescriptionField || DefaultDescriptionField;
+  const uiInline = props.uiSchema['ui:inline'];
+
+  const getInlineKeys = () => {
+    // Get array of inline keys so we can see if field exists in grid so we don't render it twice.
+    const inlineKeys: string[] = [];
+
+    uiInline &&
```

Comment:
Could have also just used a `props.uiSchema['ui:inline']?.map` instead of the check, but either still works.

Reply:
For sure and I considered that but I kind of think using a logical short circuit in jsx is more readable than using the optional chaining but I like optional chaining outside of that.

I honestly don't feel super strongly either way though.

Reply:
Neither do I, I'm alright with the short circuit. In the future, we should extend the ObjectFieldTemplateProps to include the `ui:inline` key officially rather than letting it default to `any` by the type. That or, if we know what type it is, mark it as such with an `as`

Reply:
Sounds good thanks!